### ISSUE TITLE
Minor Error in Console When Use CSS Snippet Enabled

### ISSUE THREAD

**Issue by sigrunixia** [valentine195/obsidian-admonition#267]
## Parameters
When "Use CSS Snippet for Custom Callouts" is enabled, Admonitions will generate an error within Obsidian for each "Admonition" that is created. 

## Fix
Disable "Use CSS Snippet for Custom Callouts"

## Negatives Observed
None have been observed. 

### Console Error

`Uncaught (in promise) TypeError: this.plugin.app.customCss.readCssFolders is not a function
    at d2.updateSnippet (plugin:obsidian-admonition:479:1006)`

**Comment by sigrunixia** [comment]
Looks like this is still around. Here is the specific area its "crying about.

```js
        async updateSnippet() {
            !this.plugin.data.useSnippet || (await this.plugin.app.vault.adapter.exists(this.snippetPath) ? await this.plugin.app.vault.adapter.write(this.snippetPath, this.generateCssString()) : await this.plugin.app.vault.create(this.snippetPath, this.generateCssString()),
            this.plugin.app.customCss.setCssEnabledStatus(this.plugin.data.snippetPath, !0),
            this.plugin.app.customCss.readCssFolders())
        }
```
        
The specific issue is .readCssFolders()). In main.ts, it is currently voided as a function.

**Comment by valentine195** [comment]
Fixed in 9.2.2
